packages/vue/src/StoreDesign/SDSidebar/SDSidebar.stories.js

```diff
@@ -253,6 +253,11 @@ export const Default = {
         console.log("field updated :", data);
       };
 
+      const onUpdatePreviewMode = (to) => {
+        console.log("preview mode to :", to);
+
```

Comment:
should we add any logic?

Reply:
<img width="535" alt="Screenshot 2024-05-07 at 16 21 04" src="https://github.com/hit-pay/orchid/assets/46804248/0dc9034e-1725-4b60-bd06-5064fbd68052">
for now only can update preview-mode | desktop, mobile